common/source/docs/common-foxeerf745aio.rst

```diff
@@ -0,0 +1,146 @@
+.. _common-foxeerf745aio:
+
+=========================
+Foxeer Reaper F745 AIO V2
+=========================
+
+The Foxeer Reaper F745 AIO V2 features an F7 flight controller and an integrated 45A 2-6S BLHeli_S 4-in-1 Bluejay ESC.
+
+.. note:: ArduPilot does not currently support the V3 version.
+
+.. image:: ../../../images/foxeerf745aio.png
+    :target: ../../_images/foxeerf745aio.png
+
+Where To Buy
+============
+
+`Foxeer <https://www.foxeer.com>`__
+
+Specifications
+==============
+
+-  **Processor**
+
+   -  STM32F745VG  ARM (216MHz), 1MB Flash
+   -  Integrated 4 output, BLHeli-S 45A ESC with BlueJay firmware
+   -  AT7456E OSD
+   -  16Mbit Flash for datalogging
+
+
+-  **Sensors**
+
+   -  BMI-270 IMU (Accelerometer/Gyro)
+   -  Voltage & 130A Current sensor
+
+-  **Power**
+
+   -  7.4V ~ 25V DC input power (2-6S)
+   -  5V 2.5A BEC for peripheral
+
+
+-  **Interfaces**
+
+   -  5x UARTS
+   -  5x PWM outputs, first 4 are internally connected to 4in1 40A BLHeli32 ESC. 5th used for LED PWM output
+   -  I2C port for external compass, airspeed sensor, etc.
+   -  Micro USB port
+   -  Camera input/ VTX output
+   -  DJI Goggle connector
+
+
+   **Size and Dimensions**
+
+   - 32.5mm x 32.5mm (25.5 x 25.5mm mount pattern)
+   - 8.6g
```

Comment:
This is an AIO - we should mention the ESCs as well

Reply:
it mentioned in the lead and the feature list

Reply:
what setup...the default params?

Reply:
![image](https://user-images.githubusercontent.com/2893260/206553060-2459e3d9-a8c0-4daf-af42-f93459eb9db0.png)

Reply:
How about something like this, right here:
```
 **ESCs**

   -  4 integrated BLHeli_S ESCs
   -  Firmware - BlueJay
   -  2S-4S input voltage, 45A burst current
   -  DShot 300/600
```

Reply:
okay...but 2s-6s